src/integration-test/java/uk/gov/hmcts/juror/api/AbstractIntegrationTest.java

```diff
@@ -353,4 +353,8 @@ public String getCourtJwt(String number, Set<Role> roles) {
         return createJwt("test_court_standard", number,
             UserType.COURT, roles, number);
     }
+
+    public String getSatelliteCourtJwt(String number, String... courts) {
+        return createJwt("test_court_standard", number, UserType.COURT, Set.of(), courts);
```

Comment:
Should have owner, locCode then courts.

(So that we set the loc code of the JWT correctly)

Reply:
updated number to owner, but the `createBureauJwtPayload` method uses the first court on the courts list to set as `locCode`, so it will always set the correct locCode here 😄